### PR TITLE
Improve scaleup playbook

### PR DESCRIPTION
**Pull request by detiber**
- modify evaluate host to set oo_nodes_to_config to a new variable
  g_new_nodes_group if defined rather than g_nodes_group and also skip adding
  the master when g_new_nodes_group is set.
- Remove byo specific naming from  playbooks/common/openshift-cluster/scaleup.yml
  and created a new playbooks/byo/openshift-cluster/scaleup.yml playbook.

### CODE REVIEW

**Comment by detiber** [comment]
@dak1n1 with these changes, you should be able to craft a playbooks/aws/openshift-cluster/scaleup.yml file similar to the one I created in playbooks/byo/openshift-cluster/scaleup.yml. The group names should be the only thing that changes.

This also introduces a new group variable oo_nodes_to_config which is expected to be a subset of oo_nodes_group, to only configure newly added nodes.

There are further changes that need to be made to this before it is ready for merge (the installer wrapper would need to be updated for the new path), and it would probably be good to have QE validate the change before we merge as well.

**Comment by dak1n1** [comment]
Tested on one of the internal clusters. The install finished successfully.

**Comment by detiber** [comment]
@dak1n1 thanks for being my guinea pig!

@smunilla @brenton ptal. This changes the add scaleup scenario a bit for the installer wrapper.

File: utils/src/ooinstall/openshift_ansible.py [hunk 1]
```diff
@@ -130,10 +130,10 @@ def run_main_playbook(hosts, hosts_to_run_on, verbose=False):
     inventory_file = generate_inventory(hosts_to_run_on)
     if len(hosts_to_run_on) != len(hosts):
         main_playbook_path = os.path.join(CFG.ansible_playbook_directory,
-                                          'playbooks/common/openshift-cluster/scaleup.yml')
+                                          'playbooks/byo/openshift-cluster/scaleup.yml')
     else:
         main_playbook_path = os.path.join(CFG.ansible_playbook_directory,
-                                          'playbooks/byo/config.yml')
+                                          'playbooks/byo/openshift-cluster/config.yml')
```
Comment (brenton) [inline utils/src/ooinstall/openshift_ansible.py]: Is this change intentional?  It looks like you only moved the scaleup playbook under `byo/openshift-cluster`

Comment (detiber) [inline utils/src/ooinstall/openshift_ansible.py]: @brenton yes, the playbooks/byo/config.yml plyabook simply includes playbooks/byo/openshift-cluster/config.yml and was basically only left in place because the training/docs was pointing to it already.
